platooning_strategic/config/parameters.yaml

```diff
@@ -1,5 +1,21 @@
-# The length of the trajectory in time domain, in seconds
-trajectory_time_length: 6.0
-
-# The default control plugin name
-control_plugin_name: "mpc_follower"
+maxPlatoonSize: 10
```

Comment:
Let's document these with the usual comment format including the units, ranges (where appropriate), and a description of what the values are.